docs/L1-L2 Communication/token-bridge.md

```diff
@@ -8,14 +8,16 @@ The bridges facilitate a user’s ability to conduct their transactions with the
 
 ### L1→L2 Transfer (Deposit)
 
-#### Step 1: Call The Deposit Function on L1
+#### Step 1: Call The Deposit Function on L1 and pay the deposit fee
 
-The user calls the function `deposit` (see [ERC-20 deposit](https://github.com/starkware-libs/starkgate-contracts/blob/28f4032b101003b2c6682d753ea61c86b732012c/src/starkware/starknet/apps/starkgate/solidity/StarknetERC20Bridge.sol#L10) and [ETH deposit](https://github.com/starkware-libs/starkgate-contracts/blob/28f4032b101003b2c6682d753ea61c86b732012c/src/starkware/starknet/apps/starkgate/solidity/StarknetEthBridge.sol#L10)), supplying as parameters the recipient address on StarkNet and the amount to transfer in the case of ERC-20 token. The deposit function then:
+The user calls the function `deposit` (see [ERC-20 deposit](https://github.com/starkware-libs/starkgate-contracts/blob/28f4032b101003b2c6682d753ea61c86b732012c/src/starkware/starknet/apps/starkgate/solidity/StarknetERC20Bridge.sol#L10) and [ETH deposit](https://github.com/starkware-libs/starkgate-contracts/blob/28f4032b101003b2c6682d753ea61c86b732012c/src/starkware/starknet/apps/starkgate/solidity/StarknetEthBridge.sol#L10)), supplying as parameters the recipient address on StarkNet and the amount to transfer in the case of ERC-20 token.
+
+The deposit function then:
 
 - Checks that the funds transferred are within the Alpha [limitations](./token-bridge.md#starkgate-alpha-limitations)
 - Transfers the funds from the user account to the StarkNet bridge
 - Emits a deposit [event](https://github.com/starkware-libs/starkgate-contracts/blob/28f4032b101003b2c6682d753ea61c86b732012c/src/starkware/starknet/apps/starkgate/solidity/StarknetTokenBridge.sol#L101) with the sender address on L1, the recipient address on L2, and the amount
-- Sends a message to the relevant L2 bridge with the amount to be transferred, and the recipient address as parameters. Note that, since every single bridge is dedicated to one token type, the token type doesn't have to be explicit here.
+- Sends a message to the relevant L2 bridge with the amount to be transferred, and the recipient address as parameters. Note that, since every single bridge is dedicated to one token type, the token type doesn't have to be explicit here. The deposit fee is passed along to the StarkNet Core contract, where it remains locked until the consumption of this message (for more details, see [L1→L2 message fees](./messaging-mechanism.md#l1--l2-message-fees)).
```

Comment:
```suggestion
- Sends a message to the relevant L2 bridge with the amount to be transferred, and the recipient's address as parameters. Note that, since every single bridge is dedicated to one token type, you don't need to explicitly state the token type. The deposit fee is passed along to the StarkNet Core contract, where it remains locked until the consumption of this message. For more information, see [L1→L2 message fees](./messaging-mechanism.md#l1--l2-message-fees).
```